lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.html

```diff
@@ -1,45 +1,61 @@
+>
     <div class="adf-dropdown-widget-top-labels">
-        <label class="adf-label" [attr.for]="field.id" [class.adf-left-label]="field.leftLabels">{{field.name | translate }}<span class="adf-asterisk"
-            *ngIf="isRequired()">*</span>
+        <label class="adf-label" [attr.for]="field.id" [class.adf-left-label]="field.leftLabels"
+            >{{field.name | translate }}<span class="adf-asterisk" *ngIf="isRequired()">*</span>
         </label>
     </div>
     <div>
         <mat-form-field>
             <mat-label *ngIf="getDefaultOption(list$ | async) as defaultOption">
                 {{ defaultOption.name }}
             </mat-label>
-            <mat-select class="adf-select"
-                        [id]="field.id"
-                        [(ngModel)]="field.value"
-                        [disabled]="field.readOnly"
-                        [compareWith]="compareDropdownValues"
-                        (ngModelChange)="selectionChangedForField(field)"
-                        [matTooltip]="field.tooltip"
-                        [required]="isRequired()"
-                        panelClass="adf-select-filter"
-                        (blur)="markAsTouched()"
-                        matTooltipPosition="above"
-                        matTooltipShowDelay="1000"
-                        [multiple]="field.hasMultipleValues">
+            <mat-select
+                class="adf-select"
+                [id]="field.id"
+                [(ngModel)]="field.value"
+                [disabled]="field.readOnly"
+                [compareWith]="compareDropdownValues"
+                (ngModelChange)="selectionChangedForField(field)"
+                [matTooltip]="field.tooltip"
+                [required]="isRequired()"
+                panelClass="adf-select-filter"
+                (blur)="markAsTouched()"
+                matTooltipPosition="above"
+                matTooltipShowDelay="1000"
+                [multiple]="field.hasMultipleValues"
+            >
                 <adf-select-filter-input *ngIf="showInputFilter" (change)="filter$.next($event)"></adf-select-filter-input>
 
-                <mat-option *ngFor="let opt of list$ | async"
-                            [value]="getOptionValue(opt, field.value)"
-                            [id]="opt.id">{{opt.name}}
+                <mat-option *ngFor="let opt of list$ | async" [value]="getOptionValue(opt, field.value)" [id]="opt.id">
+                    {{opt.name}}
+                </mat-option>
+                <mat-option id="readonlyOption" *ngIf="isReadOnlyType()" [value]="field.value">
+                    {{field.value}}
                 </mat-option>
-                <mat-option id="readonlyOption" *ngIf="isReadOnlyType()" [value]="field.value">{{field.value}}</mat-option>
             </mat-select>
         </mat-form-field>
-        <div *ngIf="!previewState">
+        <div *ngIf="!previewState && !field.readOnly">
```

Comment:
In this file only this place has been changed (the rest is just code style update)